### PR TITLE
Auto detect urls in fission down

### PR DESCRIPTION
**Pull request by bechurch**
## Summary
`fission down` now can auto detect urls and make the appropriate `ipns` request

Example
```bash
fission down https://ben.runfission.com
```

## Closing issues

Fixes #

## After Merge
* [ ] Does this change invalidate any docs or tutorials? _If so ensure the changes needed are either made or recorded_
* [ ] Does this change require a release to be made? Is so please create and deploy the release

### CODE REVIEW

File: library/Fission/CLI/Command/Down.hs [hunk 1]
```diff
@@ -34,22 +35,32 @@ command cfg =
       , help "The CID of the IPFS object you want to download"
       ])
 
+-- | Return an IPNS address if the identifier is a URI
+handleIPNS :: Text -> CID
+handleIPNS identifer = case URI.parseURI (Text.unpack identifer) of
+  Just uri -> do
+    let url = uri |> URI.authority |> Text.pack
+    CID ("/ipns/" <> url)
```
Comment (expede) [inline library/Fission/CLI/Command/Down.hs]: 🤔 IPNS? We're not publishing IPNS records currently, are we?

...perhaps this is meant to be a `DNSLink`?

...also possible that I'm confused 😛

Comment (bechurch) [reply library/Fission/CLI/Command/Down.hs]: @expede The `/ipns/` prefix is used to tell `ipfs.get` to retrieve the underlying cid found at the url.

https://docs.ipfs.io/guides/concepts/dnslink/

While `ipns` and `dnslink` are different concepts they still use the same prefix for the ipfs daemon. 

https://docs.ipfs.io/guides/concepts/ipns/

Comment (expede) [reply library/Fission/CLI/Command/Down.hs]: Well that's silly 😛 Cool cool, thanks for the explination!

Comment (bechurch) [reply library/Fission/CLI/Command/Down.hs]: @expede just pinging you in case you forgot to approve 😋

Comment (expede) [reply library/Fission/CLI/Command/Down.hs]: No, just got sidetracked. I need to actually run this. One sec...

**Comment by expede** [comment]
🤔 Why the `https`? Distinguishing that it's a URL, I guess?

**Comment by bechurch** [comment]
@expede pretty much! I tried to avoid it however there was no good/easy way in haskell to truthy parse something as a uri without giving a scheme.

**Comment by expede** [comment]
I see. So the logic is `if isURL then withURL ... else withCID ...` 👍

**Pull request merged**